fastlane_bot/bot.py

```diff
@@ -166,7 +166,10 @@ def get_curves(self) -> CPCContainer:
         for p in pools_and_tokens:
             try:
                 p.ADDRDEC = ADDRDEC
-                curves += p.to_cpc()
+                curves += [
```

Comment:
didn't you add the TAXTOKENS property only to mainnnet config? wouldn't this fail
for other networks?

Reply:
It is set to `[]` by default for all chains (then overridden specifically for ethereum/mainnet).